model/flow/sealing_segment.go

```diff
@@ -73,6 +73,34 @@ func (segment *SealingSegment) Lowest() *Block {
 	return segment.Blocks[0]
 }
 
+// FinalizedSeal returns the seal that seals the lowest block
+// For a valid SealingSegment, it's guaranteed to be found
+func (segment *SealingSegment) FinalizedSeal() (*Seal, error) {
```

Comment:
This method get the seal for the lowest block of the segment, it is needed because the bootstrapping process needs to add the index for the sealed block.